juniper_official/oam/ethernet-oam-lfm.rule

```diff
@@ -27,6 +27,26 @@ healthbot {
                 }
                 type string;
                 description "Discovery state for a given interface";
+                enumerate Passive Wait as 0;
+                enumerate Send Any as 1;
+                enumerate Send Local Remote as 1;
```

Comment:
We cannot have 2 values representing 1.

Also Send Local Remote and Send Local Remote Ok are good states or bad states? if they are bad states please keep it less than 1

Reply:
these are good states. updated with positive values.